code/game/objects/structures/spawner.dm

```diff
@@ -41,6 +41,8 @@
 /obj/structure/spawner/attackby(obj/item/item, mob/user, params)
 	if(scanner_taggable && is_type_in_list(item, scanner_types))
 		gps_tag(user)
+	else
+		. = ..()
```

Comment:
You should be respecting parent return value
```dm
. = ..()
if(.)
    return TRUE
if(scanner_taggable && is_type_in_list(item, scanner_types))
    gps_tag(user)
    return TRUE
```

Reply:
Actually should check that this works first - if it works that's good, if it doesn't oh well